llvm/lib/Target/RISCV/RISCVInstrInfoXCV.td

```diff
@@ -662,6 +662,8 @@ let Predicates = [HasVendorXCVelw, IsRV32], hasSideEffects = 0,
 def cv_tuimm2 : TImmLeaf<XLenVT, [{return isUInt<2>(Imm);}]>;
 def cv_tuimm5 : TImmLeaf<XLenVT, [{return isUInt<5>(Imm);}]>;
 def cv_uimm10 : ImmLeaf<XLenVT, [{return isUInt<10>(Imm);}]>;
+def cv_uimm32: Operand<XLenVT>,
```

Comment:
With the precedent set by `cv_uimm10`, `cv_tuimm5, etc., I would expect `cv_uimm32` to be isUint<32>(Imm), but its not.